plugins/mcreator-core/procedures/entity_data_integer_get.json

```diff
@@ -0,0 +1,40 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    },
+    {
+      "type": "field_data_list_selector",
+      "name": "customEntity",
+      "datalist": "customEntity"

```

Comment:
```suggestion
      "datalist": "spawnableEntity"

```

Reply:
At the moment of writing, we can't extract data parameters from external entities, so this behavior is intended.

Reply:
but it's the only way to make the entity selector work (at least from my knowledge)

Reply:
Do you have any living entities in your demo workspace? If yes and it doesn't work, can you attach it here?

Reply:
> Do you have any living entities in your demo workspace? If yes and it doesn't work, can you attach it here?

Ok

Reply:
![image](https://github.com/MCreator/MCreator/assets/16374228/dba07751-3ce4-4829-bd84-e8dfbb8e26fa)

Selector works fine for me though (current PR state)

Reply:
So is this resolved?

Reply:
@Spectrall368? Try with current branch state

Reply:
> @Spectrall368? Try with current branch state

I'll try it

Reply:
> @Spectrall368? Try with current branch state

It works